--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -104,15 +104,6 @@ int main(int argc, char *argv[])
     // We must save it here because QApplication constructor may change it
     bool isOneArg = (argc == 2);
 
-#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0)) && !defined(DISABLE_GUI)
-    // Attribute Qt::AA_EnableHighDpiScaling must be set before QCoreApplication is created
-    if (qgetenv("QT_ENABLE_HIGHDPI_SCALING").isEmpty() && qgetenv("QT_AUTO_SCREEN_SCALE_FACTOR").isEmpty())
-        Application::setAttribute(Qt::AA_EnableHighDpiScaling, true);
-    // HighDPI scale factor policy must be set before QGuiApplication is created
-    if (qgetenv("QT_SCALE_FACTOR_ROUNDING_POLICY").isEmpty())
-        Application::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::PassThrough);
-#endif
-
     try
     {
         // Create Application
